lib/innertext.js

```diff
@@ -0,0 +1,20 @@
+var Entities = require('html-entities').AllHtmlEntities
+
+var entities = new Entities()
+
+// Quick and dirty way of getting the text representation of a string of HTML,
+// basically close to what a DOM node's innerText property would be
+
+function innertext (html) {
+  // Drop everything inside <...> (i.e., tags/elements), and keep the text.
+  // Unlike browser innerText, this removes newlines, but since we're running
+  // this against heading text generated with markdown-it, we don't expect this
+  // difference to cause problems.
+  return entities.decode(html.split(/<[^>]+>/)
+    .map(function (chunk) { return chunk.trim() })
+    .filter(function (trimmedChunk) { return trimmedChunk.length > 0 })
+    .join(' ')
+  )
```

Comment:
The biggest question I have is whether this logic is sufficient. It passes our unit tests, but it feels pretty heavy-handed and lacking in…grace, I guess? Like there must be example inputs that render this invalid, but I couldn't think of them.